doc/man1/openssl-pkcs12.pod.in

```diff
@@ -277,18 +312,15 @@ can be used (see L</NOTES> section for more information). If a cipher name
 is used with PKCS#5 v2.0. For interoperability reasons it is advisable to only
 use PKCS#12 algorithms.
 
-Special value C<NONE> disables encryption of the private key and certificate.
+Special value C<NONE> disables encryption of the private key and certificates.
 
-=item B<-keyex>|B<-keysig>
+=item B<-descert>
 
-Specifies that the private key is to be used for key exchange or just signing.
-This option is only interpreted by MSIE and similar MS software. Normally
-"export grade" software will only allow 512 bit RSA keys to be used for
-encryption purposes but arbitrary length keys for signing. The B<-keysig>
-option marks the key for signing only. Signing only keys can be used for
-S/MIME signing, authenticode (ActiveX control signing)  and SSL client
-authentication, however, due to a bug only MSIE 5.0 and later support
-the use of signing only keys for SSL client authentication.
+Encrypt the certificates using triple DES, this may render the PKCS#12
+file unreadable by some "export grade" software. By default the private
```

Comment:
Perhaps there is no point mentioning the export grade software anymore. Also not using the -legacy also makes the files unreadable by such software anyway.